content/workload-reduction-toolkit/address-workload-issues/feedback-and-marking/give-feedback-effectively-without-creating-an-unmanageable-workload.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Give feedback effectively without creating an unmanageable workload
+layout: /markdown_resource.html.erb
+---
+
+# Give feedback effectively without creating an unmanageable workload
+
+<strong class="govuk-tag">Example</strong>
+
+{inset-text}
+
+## School details
+
+**School name:** Outwood Grange Academies Trust
+
+**Location:** Multiple locations across the north of England and the East Midlands
+
+**Phase:** Primary and secondary (this example is designed for secondary settings)
+
+**Number of pupils:** 25,000 pupils across 44 academies
+
+**Contact details:** Email the academy trust at <enquiries@outwood.com> 
+
+{/inset-text}
+
+<div class="govuk-grid-row dfe-width-container">
+  <div class="govuk-grid-column-full">
+    <div class="info-box">
+      <div class="info-box__corner">
+        <img src="/assets/images/bullseye.svg" alt="Bullseye icon">
+      </div>
+      <h2 class="govuk-heading-m">
+        Impact and outcomes
+      </h2>
+      <p>
+        Outwood Grange Academies Trust has created a culture where teachers give feedback effectively without creating an unmanageable workload. 
+      </p>
+      <p>
+        Thanks to their approach, teachers are innovative in how they provide feedback. They’re empowered to:
+        <ul>
+          <li>give feedback and mark in a way that best suits their subject and pupils</li>
+          <li>use a wide variety of feedback strategies such as verbal feedback and live marking</li>
```

Comment:
You're missing the closing `</ul>` here, which means your following paragraph and list will render inside this `<ul>`:

<img width="296" alt="Screenshot 2024-02-20 at 10 13 36" src="https://github.com/DFE-Digital/improve-school-workload-and-wellbeing/assets/18436946/7376aa0f-7da8-4842-8414-cb705eed1759">